stretch-auth/src/App/App.js

```diff
@@ -6,6 +6,7 @@ import CardContainer from "../CardContainer/CardContainer";
 import FormContainer from "../FormContainer/FormContainer";
 import "./App.css";
 import { secrets } from "../secrets";
+import {getSinglePokemonData} from '../apiCalls'
```

Comment:
Good job moving this over!

Reply:
Good work! Hard to keep all that organized